combustion-android-ble/src/main/java/inc/combustion/framework/service/CombustionService.kt

```diff
@@ -113,7 +115,11 @@ class CombustionService : LifecycleService() {
 
     private fun startForeground() {
         serviceNotification?.let {
-            startForeground(notificationId, serviceNotification)
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
This seems to be required as of Android 14 (https://developer.android.com/about/versions/14/changes/fgs-types-required).